syntax = "proto3";

package util_models;
import "supply-type.proto";
import "vector2d.proto";
import "player-team-id.proto";

message SupplyModel {
	int32 supply_id = 1;
	int32 player_id = 2;
	util_models.PlayerTeamId player_team_id = 3;
	util_models.SupplyType supply_type = 4;
	util_models.Vector2D position = 5;
	int32 supply_capacity = 6;
	bool is_active = 7;
}